Dockerfile

```diff
@@ -3,7 +3,7 @@ ENV PYTHONUNBUFFERED 1
 RUN mkdir /code
 ADD . /code/
 WORKDIR /code
-RUN make reqs-install-dev
 RUN pip install --upgrade pip
 RUN pip install pip-tools
-RUN pip install psycopg2-binary
```

Comment:
Was this just some legacy code here, and psycopg is installed by requirements.txt?  I think so just verifying this can be removed.

Reply:
yea, that is redundant, was put there at first to test out replace the non-binary psycopg2. will remove.